modules/ting/src/Search/TingSearchRequest.php

```diff
@@ -209,6 +216,35 @@ public function getSimpleQuery() {
   }
 
   /**
+   * The part of the query that should be interpreted as a free text query.
+   *
+   * @return string
+   *   The query fragment.
+   */
+  public function getFreeTextQuery() {
+    return $this->freeTextQuery;
+  }
+
+  /**
+   * Sets a string that should be interpeted as a free text query.
+   *
+   * The query may still contain more "advanced" parts such as a field filter.
+   *
+   * @param string $free_text_query
+   *   Any string.
+   *
+   * @return TingSearchRequest
+   *   the current query object.
+   */
+  public function setFreeTextQuery($free_text_query) {
+    if (!empty($free_text_query)) {
+      $this->freeTextQuery = $free_text_query;
+    }
```

Comment:
Is there any reason behind this check? I cannot spot is in any of the other setters.